src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/FrameHeaders.Generated.cs

```diff
@@ -191,538 +191,537 @@ public StringValues HeaderAllow
             }
         }
         public StringValues HeaderTranslate
         {
             get
             {
-                if (((_bits & 549755813888L) != 0))
+                if (((_bits & 274877906944L) != 0))
                 {
                     return _headers._Translate;
                 }
                 return StringValues.Empty;
             }
             set
             {
-                _bits |= 549755813888L;
+                _bits |= 274877906944L;
                 _headers._Translate = value; 
             }
         }
         public StringValues HeaderUserAgent
         {
             get
             {
-                if (((_bits & 1099511627776L) != 0))
+                if (((_bits & 549755813888L) != 0))
                 {
                     return _headers._UserAgent;
                 }
                 return StringValues.Empty;
             }
             set
             {
-                _bits |= 1099511627776L;
+                _bits |= 549755813888L;
                 _headers._UserAgent = value; 
             }
         }
         public StringValues HeaderOrigin
         {
             get
             {
-                if (((_bits & 2199023255552L) != 0))
+                if (((_bits & 1099511627776L) != 0))
                 {
                     return _headers._Origin;
                 }
                 return StringValues.Empty;
             }
             set
             {
-                _bits |= 2199023255552L;
+                _bits |= 1099511627776L;
                 _headers._Origin = value; 
             }
         }
         public StringValues HeaderAccessControlRequestMethod
         {
             get
             {
-                if (((_bits & 4398046511104L) != 0))
+                if (((_bits & 2199023255552L) != 0))
                 {
                     return _headers._AccessControlRequestMethod;
                 }
                 return StringValues.Empty;
             }
             set
             {
-                _bits |= 4398046511104L;
+                _bits |= 2199023255552L;
                 _headers._AccessControlRequestMethod = value; 
             }
         }
         public StringValues HeaderAccessControlRequestHeaders
         {
             get
             {
-                if (((_bits & 8796093022208L) != 0))
+                if (((_bits & 4398046511104L) != 0))
                 {
                     return _headers._AccessControlRequestHeaders;
                 }
                 return StringValues.Empty;
             }
             set
             {
-                _bits |= 8796093022208L;
+                _bits |= 4398046511104L;
                 _headers._AccessControlRequestHeaders = value; 
             }
         }
+        public StringValues HeaderContentLength
+        {
+            get
+            {
+                if (ContentLength.HasValue)
+                {
+                    return ContentLength.ToString();
```

Comment:
Changed all occurrences.

Don't think its worth storing the string; is another item to keep in sync; check if if exists/generate, clear. Is there a (non-test) scenario where its read multiple times as a string; when the numeric is available?